boomerang/src/icp_to_nicp.rs

```diff
@@ -11,9 +11,10 @@ use icrc_ledger_types::icrc1::transfer::TransferArg;
 use icrc_ledger_types::icrc2::approve::ApproveArgs;
 
 pub async fn retrieve_nicp(target: Principal) -> Result<Nat, BoomerangError> {
+    let s = get_canister_ids();
```

Comment:
is `s` the best name you could think of?

Reply:
done